envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/IncomingPermissionsTest.kt

```diff
@@ -29,7 +28,7 @@ internal class IncomingPermissionsTest : EnvoyControlTestConfiguration() {
     fun `should allow access to endpoint by authorized client`() {
         untilAsserted {
             // when
-            val response = callLocalService(endpoint = "/endpoint",
+            val response = callLocalService(endpoint = "/endpoint#fragment",
```

Comment:
Can we change it to more like real life scenario, that is change fragment to a query param?
Sending a fragment is not common (It should never be sent in request).
https://en.wikipedia.org/wiki/Fragment_identifier
> When an agent (such as a web browser) requests a web resource from a web server, the agent sends the URI to the server, but does not send the fragment. 

Sending query param is very common.